protocols/kad/Cargo.toml

```diff
@@ -21,6 +21,7 @@ log = "0.4"
 libp2p-core = { workspace = true }
 libp2p-swarm = { workspace = true }
 quick-protobuf = "0.8"
```

Comment:
Can we remove this one now?

Reply:
It's still used in `src/generated/dht/pb.rs` so probably not